recipes/configs/dev/llama2/7B_qlora.yaml

```diff
@@ -0,0 +1,91 @@
+# Config for single device QLoRA with lora_finetune_single_device.py
+# using a Llama2 7B model
+#
+# This config assumes that you've run the following command before launching
+# this run:
+#   tune download meta-llama/Llama-2-7b-hf --output-dir /tmp/Llama-2-7b-hf --hf-token <HF_TOKEN>
+#
+# To launch on a single device, run the following command from root:
+#   tune run lora_finetune_single_device --config llama2/7B_qlora_single_device
```

Comment:
will replace `lora_finetune_single_device` to `lora_finetune_fsdp2` before landing. saving CI to publish PR earlier now